src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -102,15 +102,18 @@ Copyright (c) .NET Foundation. All rights reserved.
     <LatestPatchVersionForNetCore1_0 Condition="'$(LatestPatchVersionForNetCore1_0)' == ''">1.0.10</LatestPatchVersionForNetCore1_0>
     <LatestPatchVersionForNetCore1_1 Condition="'$(LatestPatchVersionForNetCore1_1)' == ''">1.1.7</LatestPatchVersionForNetCore1_1>
     <LatestPatchVersionForNetCore2_0 Condition="'$(LatestPatchVersionForNetCore2_0)' == ''">2.0.6</LatestPatchVersionForNetCore2_0>
+    <LatestPatchVersionForNetCore2_1 Condition="'$(LatestPatchVersionForNetCore2_1)' == ''">2.1.0</LatestPatchVersionForNetCore2_1>
```

Comment:
@nguerrera @dsplaisted I added this line to have a default value when CLI is not setting LatestPatchVersionForNetCore2_1

Reply:
Actually should we set this to 2.1.1 or we should add this prop in CLI right away? And set it to 2.1.1

Reply:
My understanding is, the next prodcon build of CLI will have 2.1.1 runtime right?

Reply:
We don't need this line, it will fall through to the bundled version or the TargetFrameworkVersion if this isn't set.

Reply:
Fixed